src/AllegroFlare/Color.cpp

```diff
@@ -66,7 +66,7 @@ namespace AllegroFlare
 
 
 
-   inline ALLEGRO_COLOR operator*(float lhs, const ALLEGRO_COLOR& rhs)
+   ALLEGRO_COLOR operator*(float lhs, const ALLEGRO_COLOR& rhs)
```

Comment:
Great, thank you!  This one had me snagged.

I'm going to put this fix and the tweaks in the other two files into `master` now, so this PR can focus on the build.  (Also so the lib will actually build properly).

Reply:
✅ K, that's added here https://github.com/allegroflare/allegro_flare/commit/ac857a0458bcbccce7f815ea67cc4614ab555aeb